src/QsCompiler/Transformations/CodeTransformations.cs

```diff
@@ -10,12 +10,12 @@
 using Microsoft.Quantum.QsCompiler.Transformations.FunctorGeneration;
 using Microsoft.Quantum.QsCompiler.Transformations.SearchAndReplace;
 
-namespace Microsoft.Quantum.QsCompiler
+namespace Microsoft.Quantum.QsCompiler.Transformations
 {
     /// <summary>
     /// Static base class to accumulate the handles to individual syntax tree rewrite steps.
     /// </summary>
-    public static class CodeTransformations
+    public static class CodeTransformationExtensions
```

Comment:
Since this is a static class that only contains extension methods, this class name never actually gets used because it is an extension class or "sponsor" class. Hence I don't believe it would be a breaking change to update the name because the actual name would never appear in any code. The reason it was updated is that in my experience sponsor classes end with "Extensions", although after trying to find a standard for that online, it seems there is not a standard way of naming sponsor classes. My thinking on this is that there should be something in the name that indicates it is a sponsor class. Regardless of what specifically we do, we should change the name somehow to indicate that it is a sponsor class, as that will help make this file more readable.
[General Guidelines for Extension Methods](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/extension-methods)
[Lack of Naming Convention](https://stackoverflow.com/a/1051278/5399520)

Reply:
You can technically call extension methods by using the name of their containing class, although it's rare.

Reply:
I did this in the course of trying to make things a bit more readable for myself, however it is ultimately immaterial to the Call Graph feature, so if you are still opposed to it, it wouldn't be an issue to remove. Let me know what you think.

Reply:
I didn't realize extension methods could be called by referencing the sponsor class. I've removed the rename.